lib/ual.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+class SunbeamUAL {
+  constructor (opts) {
+    this.opts = Object.assign({
+      appName: 'Eosfinex-UAL',

```

Comment:
trailing commas is not a part of `standard.js` linting rules. Could you run `npm test` to see if it's passing, please?

Reply:
fixed